dspace-spring-rest/src/test/java/org/dspace/app/rest/BitstreamContentRestControllerIT.java

```diff
@@ -71,16 +77,8 @@
     @Before
     public void setup() throws Exception {
         super.setUp();
-        mockSolrServer = new MockSolrServer("statistics");
-        mockSolrServer.getSolrServer().deleteByQuery("*:*");
```

Comment:
Should we still clear the statistics core in one of the `@BeforeClass` methods of the super abstract classes?

Reply:
Just like we don't clear out Discovery indexes between all tests, we shouldn't have to clear out the Statistics core between every test.  If a specific test needs to do so, it can clear the Statistics core, but clearing it frequently is just wasted time.

In `BitstreamContentRestControllerIT`, the clearing of the Statistics core is unnecessary as when we are querying statistics (in individual tests) we are doing so by UUID.  And, obviously, when querying stats by UUID, we'll only be getting back statistics specific to that UUID, regardless if other stats exist.

That said, I'm not against reinserting a `@BeforeClass` in `BitstreamContentRestControllerIT` just to ensure it starts with empty stats.

Reply:
I've added a `@BeforeClass` to `BitstreamContentRestControllerIT` that now ensures it starts with clean stats.  Other (future) integration tests can do something similar if they use the Statistics core (currently only this one IT uses it)